8Knot/pages/landing/landing.py

```diff
@@ -39,7 +39,7 @@ def create_landing_hero():
                         className="landing-logo",
                     ),
                     html.H1(
-                        "Advanced open-source project analysis by Augur",
+                        "Advanced open-source project analysis",
```

Comment:
can we say "using Augur data"? I feel like this could help boost awareness of augur's existence, especially as we promote 8knot more